### PR TITLE
Fix beatmap crediting in `Beatmap Spotlights Season 9: Spring 2023 - Nostalgia` news post

### PR DESCRIPTION
**Pull request by osuZiRoX**
Beatmap is owned by me (https://osu.ppy.sh/beatmapsets/342751#fruits/757539)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by Walavouchey** [comment]
this depends entirely on the format. from what i understand, usually the host is credited, but if the intent is to credit the mapper of the diff, then this is already correct.

cc @venix12 @Hiviexd

**Comment by Hiviexd** [comment]
wala is right, in case of GDs we credit the GDer

**Comment by osuZiRoX** [comment]
This is a Collab between Deif and me, though.

**Comment by Walavouchey** [comment]
there are a few instances where collab mappers have been credited with `&` fwiw, from what i can recall

**Comment by osuZiRoX** [comment]
Confirmation from @Hiviexd or @venix12 on the format would be appreciated then

File: news/2023/2023-04-19-beatmap-spotlights-season-9-spring-2023-nostalgia.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ The Rhythm Incarnate tier threshold is picked manually based on the season's par
 - [Rita - Hajimari no Toki (Deif) \[Platter\]](https://osu.ppy.sh/beatmapsets/91485#fruits/247642), suggested by [wwwww](https://osu.ppy.sh/users/8434466)
 - [Mili - Utopiosphere (Spectator) \[Ecstasia\]](https://osu.ppy.sh/beatmapsets/187177#fruits/446769), suggested by [Kimitakari](https://osu.ppy.sh/users/4741164)
 - [Persona - Area184 (autofanboy) \[Rain184\]](https://osu.ppy.sh/beatmapsets/160955#fruits/393051), suggested by [SadEgg](https://osu.ppy.sh/users/10278243)
-- [Hylian Lemon - Foresight Is for Losers (Deif) \[Collab Rain\]](https://osu.ppy.sh/beatmapsets/342751#fruits/757539), suggested by [wonjae](https://osu.ppy.sh/users/5032045)
+- [Hylian Lemon - Foresight Is for Losers (ZiRoX) \[Collab Rain\]](https://osu.ppy.sh/beatmapsets/342751#fruits/757539), suggested by [wonjae](https://osu.ppy.sh/users/5032045)
```
Comment (Walavouchey) [inline news/2023/2023-04-19-beatmap-spotlights-season-9-spring-2023-nostalgia.md:275]:
```suggestion
- [Hylian Lemon - Foresight Is for Losers (ZiRoX & Deif) \[Collab Rain\]](https://osu.ppy.sh/beatmapsets/342751#fruits/757539), suggested by [wonjae](https://osu.ppy.sh/users/5032045)
```

**Review by Walavouchey — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**